libraries/JGUZDV.Blazor.StateManagement/test/UnitTest1.cs

```diff
@@ -0,0 +1,11 @@
+namespace JGUZDV.Blazor.StateManagement.Tests
```

Comment:
Wenn wir nichts zu testen finden, kann die Datei auch weg.
Wir sollten aber einen Blick in bUnit werfen, um auch bei Blazor nicht blank zu sein.

Reply:
Ja doch das kann man schon testen, hab mal einen hinzugefügt